### PR TITLE
[MXNET-906] ONNX export/import: Selu

### PR DESCRIPTION
**Pull request by vandanavk**
## Description ##
Add support for ONNX import/export of Selu activation

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- Add Selu import/export
- Add tests for selu

## Comments ##
- alpha and lambda are fixed in MXNet based on the paper arxiv:1706.02515 (Eqn 14)

### CODE REVIEW

**Comment by vandanavk** [comment]
@mxnet-label-bot [pr-awaiting-review, ONNX]

File: tests/python-pytest/onnx/import/test_cases.py [hunk 1]
```diff
@@ -98,6 +98,7 @@
     'test_MaxPool',
     'test_PReLU',
     'test_ReLU',
+    'test_selu_default',
```
Comment (anirudhacharya) [inline tests/python-pytest/onnx/import/test_cases.py]: why is `test_selu` being excluded?

File: python/mxnet/contrib/onnx/onnx2mx/_import_helper.py [hunk 2]
```diff
@@ -75,6 +75,7 @@
     'LeakyRelu'         : leaky_relu,
     'Elu'               : _elu,
     'PRelu'             : _prelu,
+    'Selu'              : _selu,
```
Comment (anirudhacharya) [inline python/mxnet/contrib/onnx/onnx2mx/_import_helper.py]: will there be a namespace collision with selu? why _selu?

Comment (vandanavk) [reply python/mxnet/contrib/onnx/onnx2mx/_import_helper.py]: followed the same naming convention as _elu, _prelu.

File: tests/python-pytest/onnx/import/test_cases.py (hunk 1 above)
Comment (vandanavk) [reply tests/python-pytest/onnx/import/test_cases.py]: alpha and lambda are fixed based on the paper arxiv:1706.02515 (Eqn 14). for self-normalizing property of SNNs, this fixed alpha and lambda value is required.
the tests in ONNX for selu were added as "testing math operators" and different alpha/lambda values were provided.

Comment (anirudhacharya) [reply tests/python-pytest/onnx/import/test_cases.py]: I see, so alpha and lambda cant be passed as parameters to `LeakyRelu` which has `selu` as an activation type. That might be a bug.

Comment (vandanavk) [reply tests/python-pytest/onnx/import/test_cases.py]: update based on offline discussions - MXNet's implementation fixed these values based on the recommendation in the paper, so alpha and lambda will continue to be the default values and not configurable parameters to LeakyRelu. following this, only test_selu_default will be added as a test for ONNX import/export.

**Comment by vandanavk** [comment]
Adding @Roshrini @zhreshold for review

**Comment by vandanavk** [comment]
Is this PR good to go @zhreshold @anirudhacharya ?

**Pull request merged**